background.js

```diff
@@ -2,6 +2,7 @@ const defaultOptions = {
   pinned: false,
   audible: false,
   time: 1800000,
+  timeToClose: 3600000,
```

Comment:
```suggestion
  timeToClose: 0,
```

I'd prefer to keep the current functionality of only suspending the tabs, not closing them :)

Reply:
Oh you are totally right!